mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1115,4 +1145,57 @@
+    <!-- Title text shown for the migration screen to the new browser -->
+    <string name="migration_title">Welcome to an all-new Firefox</string>
+    <!-- Description text followed by a list of things migrating (e.g. Bookmarks, History) -->
+    <string name="migration_description">A completely redesigned browser awaits, with improved performance and features to help you do more online.\n\nPlease wait while we update Firefox with your</string>
+    <!-- Text on the disabled button while in progress -->
+    <string name="migration_updating_app_button_text">Updating %s…</string>
+    <!-- Text on the enabled button -->
+    <string name="migration_update_app_button">Start %s</string>
+    <!-- Accessibility description text for completed migration item (e.g. Bookmarks, History) -->
+    <string name="migration_icon_description">Migration status: %s</string>
+
+    <!-- Site Permissions Preferences -->
+    <!-- Preference for reducing image sizes, bandwidth and platform-level optimizations -->
+    <string name="preference_optimize">Optimize</string>
+    <!-- Label summary to explain how the optimize preference works -->
+    <string name="preference_optimize_summary">Lower image quality, throttle streaming bandwidth, and platform-level optimizations</string>
+    <!-- Preference for applying recommend rules to all sites -->
+    <string name="preference_recommended_settings">Use recommended settings</string>
+    <!-- Label summary to explain how the recommended settings work -->
+    <string name="preference_recommended_settings_summary"><![CDATA[
+        <b>Blocked</b>
+
+        <br/> Ads, autoplay sound and video (block playing media with sound), cookies (block third-party trackers cookies), trackers (allow some trackers), pop-up, website redirects. <br/><br/> <b>Ask to allow</b>
+
+        <br/> Camera, location, microphone and notification. <br/><br/> <b>Allowed</b>
+
+        <br/> DRM audio and video, JavaScript, cache and site data, images.]]></string>
+    <!-- Preference for applying custom rules to all sites -->
+    <string name="preference_custom_settings">Use custom settings</string>
+    <!-- Preference category for feature phone permissions likes Camera,Microphone, Location ... etc -->
+    <string name="preference_category_phone_feature">Phone Feature</string>
+    <!--Label that indicates that a user hasn't select a value for a site permission-->
+    <string name="phone_feature_no_decision">No Decision</string>
+    <!-- Alternative explanation label that is shown when a permissions like (camera,location and microphone) is required, this indicate to the user how to enable the permission via Android settings %1$s indicate the name of the permission (camera,location and microphone)  -->
+    <string name="phone_feature_blocked_by_android_explanation"><![CDATA[
+        To allow it: <br/><br/> 1. Go to Android Settings <br/><br/>2. Tap <b>Permissions</b> <br/><br/> 3. Toggle <b>%1$s</b> to ON
+        ]]></string>
+
```

Comment:
Again, this allows localizers to change layout. Not sure we want this

Reply:
Thanks for raising this! Since this is an existing issue in other strings, is it reasonable that I land this fix as a separate ticket? I'd love to clean all of these up at once 😄

Reply:
Issue filed [here](https://github.com/mozilla-mobile/fenix/issues/7965) regarding this.

Reply:
Thank you! 😃

Reply:
Commented on Slack, but just in case :)
I'll wait until that issue is fixed to merge this PR and expose the strings on Pontoon, because if I merge already now, localizers will work on these strings and their translations will be lost once the new strings arrive.
thanks!